docs/deployment-examples.md

```diff
@@ -0,0 +1,331 @@
+# Example Deployment Methods
+There are a number of methods to deploy the OpenMetrics exporter depending on your environment. In this document we explore some of the common features and deployment methods and by providing walk through examples.
```

Comment:
`in your environment`.

remove the `and` after delpoyment methods

Reply:
Resolved in commit 4010cfcac1c820121ac3f1816ef1ed4180d0595b